provision-sm-md-lg/README.md

```diff
@@ -0,0 +1,71 @@
+# Provision VM Service Small Medium Large Example
+
+This example shows creating a provisioning dialog that modifies provisioning parameters
+
+## Architecture
+
+- `provision-vm-small-med-large.asl`: This workflow is run for every vm provisioned and is responsible for modifying the vm size. This will be used as a ManageIQ Service's Provisioning Entry Point.
+
+Set the ManageIQ provisioning service entry point to this script. This sample workflow is run for every provisioned Virtual Machine (VM), and modifies the VM size.
+
+## Add a Workflow Repository
+
+1. **Click Automation** > **Embedded Workflows** > **Repositories**
```

Comment:
Click should not be in bold. Anything that is inside the "**"  **text-here** will be rendered as bold.
example
Click **Automation** > **Click Automation** > **Repositories**